src/soft-play.js

```diff
@@ -6,9 +6,59 @@ let children = 0
 // the adults and children variables defined above.
 // Start with the occupancy function.
 
+function occupancy() {
+  const total = { adults: adults, children: children }
+  return total
+}
+
+console.log(occupancy())
+
+const totalEntered = { adults: 0, children: 0 }
+
+function newPeople(a, c) {
```

Comment:
what does a and c stand for? this is highly unreadable

Reply:
Hi Carlo. I have now tweaked the name of the values for easier reading of the code. This has been pushed back to gitHub. Hopefully you will be able to see this now.